### PR TITLE
Integration

### PR DESCRIPTION
**Pull request by wpdunk**

### CODE REVIEW

File: src/components/poseWrapper/index.js [hunk 1]
```diff
@@ -1,30 +1,64 @@
...
+import StyledChecklist from "../checklist";
 
 class PoseWrapper extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      result: null,
+      activePose: null
+    };
+    this.handlePose = this.handlePose.bind(this);
```
Comment (punchcafe) [inline src/components/poseWrapper/index.js]: would be great to know what this is doing EXACTLY

File: src/components/poseWrapper/index.js [hunk 2]
```diff
@@ -1,30 +1,64 @@
...
+  }
+
+  componentDidMount() {
+    this.runPosenet();
+  }
+
+  handlePose(wrappedPose) {
```
Comment (punchcafe) [inline src/components/poseWrapper/index.js]: curious as to what's happening here. is it overwriting?

File: src/components/poseWrapper/index.js [hunk 3]
```diff
@@ -1,30 +1,64 @@
...
+    var result = wrappedPose.isMountainPose();
+    this.setState({
```
Comment (punchcafe) [inline src/components/poseWrapper/index.js]: B   I   N .  D with me william

File: src/components/poseWrapper/index.js (hunk 1 above)
Comment (wpdunk) [reply src/components/poseWrapper/index.js]: as in the implemented method handlePose we’re dealing with the component’s state object we need to make sure to bind those methods to this by adding this beautiful line of code to the constructor

**Pull request merged**